src/components/ProjectList.js

```diff
@@ -1,78 +1,35 @@
-import React, { useEffect, useState } from 'react'
+import React from 'react'
 import styled from 'styled-components'
-import { v4 as uuidv4 } from 'uuid'
-import addButtonShadowTop from '../assets/addButtonShadowTop.svg'
-import AddNewProjectTab from './AddNewProjectTab.js'
-import LogoHeader from './LogoHeader.js'
 import Headline from './Headline'
 import NavigationBar from './NavigationBar'
 import ProjectListItem from './ProjectListItem'
 
-export default function ProjectList() {
-  const [projectList, setProjectList] = useState([])
-  const [showAddNewProjectTab, setShowAddNewProjectTab] = useState(false)
-
-  useEffect(() => {
-    setProjectList(JSON.parse(localStorage.getItem('projects') || '[]'))
-  }, [])
-
-  useEffect(() => {
-    localStorage.setItem('projects', JSON.stringify(projectList))
-  }, [projectList])
-
+export default function ProjectList({ projectList, updateProjectData }) {
   return (
     <>
-      <LogoHeader />
-      {showAddNewProjectTab ? (
-        <AddNewProjectTab
-          onSubmit={addToProjectList}
-          onBack={() => setShowAddNewProjectTab(false)}
+      <StyledTab>
+        <Headline
+          headlineText="Projects"
+          textColor={'var(--copper-ultralight)'}
         />
-      ) : (
-        <StyledTab>
-          <Headline
-            headlineText="Projects"
-            textColor={'var(--copper-ultralight)'}
-          />
-          {projectList.length <= 0 && (
-            <p className="noProjectsYet">
-              Click the button below to add your first project!
```

Comment:
Bei Vercel bricht es nach dem "to" um. Du könntest versuchen, das "to" auf die untere Zeile zu ziehen. Dann passt es syntaktisch besser :)